File: scripts/restore-clientes.mjs
import { execSync } from 'child_process';
import path from 'path';

const projectPath = '/vercel/share/v0-project';
process.chdir(projectPath);

try {
  console.log('[v0] Restaurando arquivo original do git...');
  const result = execSync('git checkout app/pagamentos/clientes/page.tsx', {
    encoding: 'utf-8',
    stdio: 'pipe',
  });
  console.log('[v0] Resultado:', result);
  console.log('[v0] Arquivo restaurado com sucesso!');
} catch (error) {
  console.error('[v0] Erro ao restaurar:', error instanceof Error ? error.message : String(error));
}

File: scripts/restore-clientes.py
#!/usr/bin/env python3
import subprocess
import os

os.chdir('/vercel/share/v0-project')

try:
    print('[v0] Restaurando arquivo original do git...')
    result = subprocess.run(
        ['git', 'checkout', 'app/pagamentos/clientes/page.tsx'],
        capture_output=True,
        text=True,
        check=True
    )
    print('[v0] Stdout:', result.stdout)
    print('[v0] Arquivo restaurado com sucesso!')
except subprocess.CalledProcessError as e:
    print(f'[v0] Erro ao restaurar: {e.stderr}')
except Exception as e:
    print(f'[v0] Erro: {str(e)}')
